Changelog.md

```diff
@@ -11,6 +11,7 @@
 - Added the `gtin` field to the products and master product variants tables
 - Fixed the missing float cast of the order item's price field
 - PHP 8.4 deprecation notice fixes
+- Added ETA fields to Shipping Methods
```

Comment:
It's a sad story, but with VFW we need to add the Changelog entry both to the Module's changelog (`src/Shipment/Changelog.md`) and to the "global" changelog, this file :point_up: 

This so because each module has a standalone subtree split with its own, scoped changelog: https://github.com/vanilophp/shipment/blob/4.2.0/Changelog.md